scrapi/harvesters/mit.py

```diff
@@ -0,0 +1,891 @@
+
+"""Harvests MIT DSpace metadata for ingestion into the SHARE service
+
+More information available here:
+https://github.com/CenterForOpenScience/SHARE/blob/master/providers/edu.mit.md
+
+Example metadata URL: http://dspace.mit.edu/oai/request?verb=ListRecords&metadataPrefix=oai_dc&from=2014-09-28
+"""
+
+
+from __future__ import unicode_literals
+
+from scrapi.base import OAIHarvester
+
+
+class MITHarvester(OAIHarvester):
+    long_name = 'MIT'
+    short_name = 'mit'
+    base_url = 'http://dspace.mit.edu/oai/request'
+    property_list = [
```

Comment:
Is there a reason why approved sets is a python property, but property_list isn't?